ci/test.yml

```diff
@@ -18,6 +18,7 @@
     - docker login $CI_DEPENDENCY_PROXY_SERVER -u $CI_DEPENDENCY_PROXY_USER -p $CI_DEPENDENCY_PROXY_PASSWORD
   script:
     - yarn install --pure-lockfile --cache-folder .yarn --prefer-offline
+    - yarn workspace @trezor/utils build:lib
```

Comment:
one last from me!
could you pleass add this line
```
- yarn workspace @trezor/utils build:lib
```
also to `.e2e transport` and `.e2e blockchain-link`? 

there are also imports from this lib.

Reply:
Done!